### PR TITLE
Refactor TeamColorizer into a Python library

### PR DESCRIPTION
**Pull request by Slayer95**
Reduces subprocess spamming in ``wmlunits``.

### CODE REVIEW

File: data/tools/TeamColorizer [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import sys, argparse, subprocess
```
Comment (ProditorMagnus) [inline data/tools/TeamColorizer:3]: why import subprocess?

File: data/tools/TeamColorizer [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    exit_code = colorize(
+        namespace.color,
+        namespace.input_file,
+        namespace.output_file,
+        magick=namespace.magick,
+        method=namespace.method,
+        hex=namespace.hex,
+        namespace=namespace
+    )
+    if not dry_run and exit_code != 0:
```
Comment (ProditorMagnus) [inline data/tools/TeamColorizer:90]: previously dryrun

Comment (Slayer95) [reply data/tools/TeamColorizer:90]: Good catch

File: data/tools/TeamColorizer (hunk 1 above)
Comment (Slayer95) [reply data/tools/TeamColorizer:3]: Leftover

**Comment by CelticMinstrel** [comment]
Why does this use ImageMagick, anyway? It seems to me that it'd make more sense to use Pillow.

**Comment by Slayer95** [comment]
Probably because this was written before Pillow was a thing.

**Comment by Elvish-Hunter** [comment]
Although the script was written before Pillow was made, back then PIL already existed (and Pillow is a fork of it). I suppose that there are some good reasons to not get rid of ImageMagick support (perhaps better image compression?), but it'd be nice to add support for Pillow as well (in another pull request).
That said, I don't see anything really wrong with the code, so I think it can be merged.

**Pull request merged**